lib/js/components/ui-char-groups.mjs

```diff
@@ -476,13 +475,13 @@ export class UIUpdateListener extends _BaseComponent {
 
 export class UICharGroupContainer extends _BaseTypeDrivenContainerComponent {
     constructor(widgetBus, _zones, injectable, propertyRoot, label) {
-        // run super first, so we can use `this` in the widgets definition.
-        const localZoneElement = widgetBus.domTool.createElement('fieldset', {'class': 'ui_char_group_container'})
-          , zones = new Map([..._zones, ['local', localZoneElement]])
+        const localZoneElement = widgetBus.domTool.createElement('div', {'class': 'ui_char_group_container'})
+          , contentsZoneElement = widgetBus.domTool.createElement('div')
+          , zones = new Map([..._zones, ['local', localZoneElement], ['contents', contentsZoneElement]])
           ;
         super(widgetBus, zones);
         this._insertElement(localZoneElement);
-        collapsibleMixin(localZoneElement, 'legend');
+        this._contentsZoneElement = contentsZoneElement;
```

Comment:
Don't  keep this as an instance property, just pass it as argument directly into `_defineWidgets`.

Reply:
NOTE ALSO: `localZoneElement` as argument of `_defineWidgets` seems to be unused, so it can be replaced by `contentsZoneElement`